### ISSUE TITLE
France: still wrong numbers of confirmed cases (April 16)

### ISSUE THREAD

**Issue by alfkoehn** [CSSEGISandData/COVID-19#2238]
Number in database here: 133470

Number in other data sources [1,2]: 106206

27264 cases too much, probably due to also including **potential** cases (and not only **confirmed** cases), see e.g. the discussion [3] and an official statement [4], where it was promised to fix that (but nothing has happened since then).

[1] https://github.com/opencovid19-fr/data/blob/master/ministere-sante/2020-04-14.yaml
[2] https://dashboard.covid19.data.gouv.fr/
[3] https://github.com/CSSEGISandData/COVID-19/issues/2005
[4] https://github.com/CSSEGISandData/COVID-19/issues/2094

**Comment by servoz** [comment]
I can't understand why nothing is being done when there are several pull requests supposed to fix several tickets open on this subject !

**Comment by alfkoehn** [comment]
And don't forget they promised to fix it (nothing has happened since then).

**Comment by rozierguillaume** [comment]
Please accept the merge request from Yqnn!

**Comment by Aureuum** [comment]
@alfkoehn Yes, the time series data for France since 4/4 are in large excess (about 20k-30k) compared to the correct official data. The probable cause is that the jhu.edu site itself shows the wrong data, and then everybody copies it.

**Comment by Aureuum** [comment]
Since they don't seem to fix it, you can just replace your data with this:
4/1/20,4/2/20,4/3/20,4/4/20,4/5/20,4/6/20,4/7/20,4/8/20,4/9/20,4/10/20,4/11/20,4/12/20,4/13/20,4/14/20,4/15/20
56989,59105,64338,68605,70478,74390,78167,82048,86334,90676,93790,95403,98076,103573,106206

**Comment by servoz** [comment]
thanks !

**Comment by Vanuan** [comment]
what's going on?
![image](https://user-images.githubusercontent.com/361985/79515681-fb987380-8051-11ea-9af0-5387da4c909a.png)
